### ISSUE TITLE
bluetooth headphones paired but not connected

### ISSUE THREAD

**Issue by KingBubu** [blueman-project/blueman#492]
Hi all,

I recently bought the Bose® SoundLink® around-ear wireless headphones II and tried to connect them to my ThinkPad T440s.

blueman: v2
BlueZ: v5
Distribution: Ubuntu 15.10
Desktop environment: Unity

https://gist.github.com/KingBubu/319cf2f81e3b2d64c2f4c952dfdb50ed

I read the debian and archlinux wiki pages and also tried to connect via bluetoothctl. Here is what `systemctl status bluetooth` gives me:

`Unable to register GATT service with handle 0x0001 for device 08:DF:1F:C3:32:02:`

Any help is greatly appreciated.

Thanks!

**Comment by cschramm** [comment]
BlueZ' GATT API is an experimental feature. You have to start bluetoothd with `-E`.

**Comment by KingBubu** [comment]
I upgraded my system to Ubuntu 16.04 and successfully connected my headphones to it. In Sound Settings I selected the A2DP profile and everything seemed to be perfect. However when there is no music playing or only a little bit I distinctly hear a background noise. Is this normal? 

In /etc/pulse/daemon.conf I uncommented the line "enable-lfe-remixing = yes" in order for my external soundcard to work correctly. Could this be an issue?
